.github/workflows/sync-release-to-linear.yml

```
@@ -0,0 +1,317 @@
+          PR_BODY="$(jq -r '.pull_request.body // ""' "$GITHUB_EVENT_PATH")"
+          PR_URL="$(jq -r '.pull_request.html_url // ""' "$GITHUB_EVENT_PATH")"
+          HEAD_SHA="$(jq -r '.pull_request.head.sha // ""' "$GITHUB_EVENT_PATH")"
+
+          VERSION="$(printf '%s\n%s\n' "$PR_TITLE" "$PR_BODY" | grep -Eo 'v[0-9]+([.][0-9]+){0,3}([-.][A-Za-z0-9._]+)?' | head -n 1 || true)"
+          if [ -z "$VERSION" ]; then
+            VERSION="release"
+          fi
+
+          LINEAR_TITLE="[Release ${VERSION}] Promote development -> master (GH PR #${PR_NUMBER})"
+          LINEAR_DESCRIPTION="$(cat <<EOF
+Automated release tracking for development -> master promotion.
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)